kill-counter.lic

```diff
@@ -0,0 +1,99 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#kill-counter
+=end
+
+unless HAVE_GTK
+  respond
+  respond 'error: ruby-gtk bindings are not installed or failed to load'
```

Comment:
Have you considered a `nowindow` option, similar to how `status-monitor` works? Some users have problems with GTK.